Makefile

```diff
@@ -111,6 +112,21 @@ build-query-linux:
 build-collector-linux:
 	CGO_ENABLED=0 GOOS=linux installsuffix=cgo go build -o ./cmd/collector/collector-linux ./cmd/collector/main.go
 
+.PHONY: docker
+docker: build_ui build-agent-linux build-collector-linux build-query-linux
+	cp -r jaeger-ui-build/build/ cmd/query/jaeger-ui-build
```

Comment:
That's because the Dockerfile is within the cmd/query , and Docker can only build stuff that is on its own directory or any sub directory, but not in any parent directory (for security reasons I suppose). So, we need to have a copy there.